systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectS2IST.java

```diff
@@ -234,7 +231,7 @@ void testSecretsWithKafkaConnectS2IWithTlsAndScramShaAuthentication() {
             .withKafkaUsername(userName)
             .build();
 
-        String kafkaConnectS2IPodName = kubeClient().listPods("type", "kafka-connect-s2i").get(0).getMetadata().getName();
+        String kafkaConnectS2IPodName = kubeClient().listPods("type", KafkaConnectS2I.RESOURCE_KIND).get(0).getMetadata().getName();
```

Comment:
`listPods("type", "kafka-connect-s2i")` vs `String RESOURCE_KIND = "KafkaConnectS2I";`
je to ok?

Reply:
ano je, podivej se kde se to vyuziva